doc/developer/design/20210714_reclocking.md

```diff
@@ -0,0 +1,98 @@
+# Reclocking
+
+## Summary
+
+"Reclocking" a stream of data is (for the purposes of this document) translating its events from one time domain to another time domain.
```

Comment:
Is "time domain" rigorously defined anywhere?

Reply:
No. I think the closest thing may be "timeline" from that design doc, but I didn't want to presume they were the same thing. It is possible that "time domain" should everywhere be replaced by "timeline", except that these may be things that do not have an actual `Timeline` associated with them (but would, if they were to be used in MZ with their native timestamp).

Reply:
"time domain" is a term already used by the coordinator at https://github.com/MaterializeInc/materialize/blob/196217a455f1685267c94add415e77f11a4704ba/src/coord/src/coord.rs#L2203 which was a term from https://github.com/MaterializeInc/materialize/issues/4660 which claims that it is "A phrase @frankmcsherry has used several times". We should standardize this term. The coordinator is happy to rename it to something else if we want. I don't care beyond making sure it only means one thing (both to the code, and by us).

Reply:
I think morally it is the same thing here, and I'll bring this language into alignment. Or, .. maybe before doing that I should double check that it is what I think it is, but then I'll bring it in to alignment.